docs/src/pages/[platform]/components/table/TablePropControls.tsx

```diff
@@ -59,7 +59,7 @@ export const TablePropControls: TablePropControlsInterface = ({
       <option value="large">large</option>
     </SelectField>
     <SelectField
-      id="variation"
+      id="variationProp"
```

Comment:
I guess we need to update id for `size` as well?

Reply:
Oops good catch! Updated.